app/components/trn_details_component.html.erb

```diff
@@ -48,6 +54,7 @@
   rows.push(previous_name_row) if @trn_request.previous_name?
   rows.push(date_of_birth_row)
   rows.push(ni_row)
+  rows.push(ask_trn_row) if FeatureFlag.active?(:ask_for_trn)
```

Comment:
This row shouldn't be visible if the user isn't in the identity flow.

Could we check for that as well? The attribute is `.from_get_an_identity` on a `TrnRequest`.